### PR TITLE
Redesign trigger/condition/action overflow menus

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Needs: #13444

Trigger/Condition/Action menus:

CleanShot 2022-08-23 at 18 51 34

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by frenck** [comment]
Rebased onto `dev` to get it on top of #13444

**Comment by frenck** [comment]
Drafting it, going to change the order probably

**Comment by frenck** [comment]
Some small tweaks and cleanups.

Trigger/Condition/Action menus:

CleanShot 2022-08-23 at 18 51 34

File: src/panels/config/automation/action/ha-automation-action-row.ts [hunk 1]
```diff
@@ -187,43 +198,120 @@ export default class HaAutomationActionRow extends LitElement {
               .label=${this.hass.localize("ui.common.menu")}
               .path=${mdiDotsVertical}
             ></ha-icon-button>
-            <mwc-list-item>
+            <mwc-list-item
+              .label=${this.hass.localize(
```
Comment (steverep) [inline src/panels/config/automation/action/ha-automation-action-row.ts]: Same comment from #13457 applies here with the added caveat that `label` is not a valid property.  Let's get rid of these.

**Review by steverep — CHANGES_REQUESTED** [review]

File: src/panels/config/automation/action/ha-automation-action-row.ts (hunk 1 above)
Comment (frenck) [reply src/panels/config/automation/action/ha-automation-action-row.ts]: @steverep I think your comment is based on an older commit.

**Comment by frenck** [comment]
Converting to draft again, going to simply the menu's in wording.

**Review by steverep — DISMISSED** [review]
Approving with the caveat that my opinion on the applicability of tiny icons is undefined 👨‍🦯

**Pull request merged**